pandas-stubs/core/arrays/integer.pyi

```diff
@@ -1,26 +1,27 @@
+import numpy as np
+
+from pandas._libs.missing import NAType
+
 from pandas.core.dtypes.base import ExtensionDtype as ExtensionDtype
 
 from .masked import BaseMaskedArray
 
-_type = type
-
 class _IntegerDtype(ExtensionDtype):
```

Comment:
the class is "hidden", so I don't think we should include it in typing.

Reply:
It is used in inheritance for other types, e.g., class `Int8Dtype(_IntegerDtype)`.